### ISSUE TITLE
How should this theme be licensed?

### ISSUE THREAD

**Issue by clausmullie** [standard-for-public-code/jekyll-theme#7]
Should the license change from EUPL to CC0 on this? Or fine to leave it as is?

**Comment by jgroenen** [comment]
I heard CC(0) is not compatible with open source licenses (but I'm not an expert)?

**Comment by jgroenen** [comment]
No, never mind: https://wiki.creativecommons.org/wiki/CC0_FAQ#:~:text=Yes%2C%20CC0%20is%20suitable%20for,software%20licenses%2C%20including%20the%20GPL.
